### ISSUE TITLE
tax inheritance only works with 3 sub-accounts

### ISSUE THREAD

**Issue by s0enke** [superwerker/aws-organizations-vending-machine#18]
since checkbox order matters

example for not working account: `ovm-374a5779`

**Comment by s0enke** [comment]
fixed by fbef6d89684a9bbae7046f25a80e38af3aff804d

**Comment by s0enke** [comment]
Script to process all closed accounts again:
```
import boto3
import json

ddb = boto3.client("dynamodb")
sqs = boto3.client("sqs")

accounts = ddb.execute_statement(Statement="select account_name,account_email from account.account_status where account_status='CLOSED'")['Items']

for account in accounts:
    account_name = account['account_name']['S']
    try:
        account_email = account['account_email']['S']
    except KeyError:
        account_email = 'XXX'.format(account_name.split('-')[1])
    queue_body = {
        'account_name': account_name,
        'account_email': account_email,
    }
    print(queue_body)

    sqs.send_message(
        QueueUrl='XXX',
        MessageBody=json.dumps(
            queue_body
        )
    )


```
